actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "pathname"
+require "json"
+
+module ActionText
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path("templates", __dir__)
+
+      def install_javascript_dependencies
+        template "#{__dir__}/../../../../../railties/lib/rails/generators/rails/app/templates/bin/yarn.tt",
```

Comment:
Don't think this will work once Action Text get's packaged as a gem. That file doesn't ship within it.

Maybe we can copy it when building the gem? What do you think @y-yagi?